### PR TITLE
Upgrade Symfony dependencies to 3.0

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     ],
     "require": {
         "php": ">=5.4.8",
-        "symfony/event-dispatcher": "~2.3",
-        "symfony/options-resolver": "~2.3",
+        "symfony/event-dispatcher": "^2.3||^3.0",
```
Comment (ddeboer) [inline composer.json]: Also switching to caret here, as it’s the [recommended operator](https://getcomposer.org/doc/articles/versions.md#caret).

**Comment by dbu** [comment]
hm, looks like composer is having a hard time making sense of this. or are we doing the mistake?

**Comment by ddeboer** [comment]
Ah, the problem is not our requirement format, but the fact that some Symfony 3.0 packages require 5.5.

**Comment by dbu** [comment]
yeah. but do we force symfony 3 here? i think composer should realize we
are on a too old php and pick an older symfony, no?

however i am ok if we just force it to 2.8 on php 5.4

**Comment by ddeboer** [comment]
For one or another reason ``prefer-lowest` only started behaving properly (installing 2.3 on PHP 5.4) when I split `before_install` off from `install`. Anyway, it works now. :wink:

**Comment by dbu** [comment]
great! should we do this pull request against the 1.3 branch so that we can do a dot release that supports symfony 3? 2.0 might still take a moment until its ready for release.

**Comment by ddeboer** [comment]
Or just merge this to master and cherry-pick it into the 1.3 branch?

**Comment by dbu** [comment]
i'd rather do it the other way round, to avoid merge conflicts.

**Comment by ddeboer** [comment]
Okay, closed this and moved to #259.

**Pull request closed without merging**